### PR TITLE
Add Banner Notification as extension point

### PR DESCRIPTION
**Pull request by mariam21arauj**
Addition of Banner Notification as extension point. 

## Description

Addition of Banner Notification as extension point to the Adobe Commerce Sample apps to allow users to customize banner notifications for custom mass actions and custom order view buttons 

## Motivation and Context

Simplifies and enhances the user's usage of the Admin UI SDK extension points.

## How Has This Been Tested?

- Tested with local Commerce 2.4.6 instance configuration for the Admin UI SDK 1.4.0 
- Deployed and ran app on the Adobe Developer Console

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: admin-ui-sdk/banner-notification/custom-mass-actions/web-src/src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  })
+
+  return (
+      <ErrorBoundary onError={onError} FallbackComponent={fallbackComponent}>
+          <HashRouter>
+              <Provider theme={lightTheme} colorScheme={'light'}>
+                  <Routes>
+                      <Route index element={<ExtensionRegistration runtime={props.runtime} ims={props.ims} />} />
+                      <Route path={'order-mass-action'} element={<OrderMassAction runtime={props.runtime} ims={props.ims} />} />
+                      <Route path={'product-mass-action'} element={<ProductMassAction runtime={props.runtime} ims={props.ims} />} />
```
Comment (asalloum5) [inline admin-ui-sdk/banner-notification/custom-mass-actions/web-src/src/components/App.js:38]:
```suggestion
                      <Route index element={<ExtensionRegistration runtime={props.runtime} ims={props.ims} />} />
```

File: admin-ui-sdk/banner-notification/custom-mass-actions/web-src/src/components/ExtensionRegistration.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                                successMessage: 'Custom success message',
+                                errorMessage: 'Custom error message'
+                            }
+                        ]
+                    }
+                }
+            }
+        }
+    })
+}
```
Comment (asalloum5) [inline admin-ui-sdk/banner-notification/custom-mass-actions/web-src/src/components/ExtensionRegistration.js:47]: Missing empty line end of file

File: admin-ui-sdk/banner-notification/order-view-button/web-src/src/components/App.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  })
+
+  return (
+      <ErrorBoundary onError={onError} FallbackComponent={fallbackComponent}>
+          <HashRouter>
+              <Provider theme={lightTheme} colorScheme={'light'}>
+                  <Routes>
+                      <Route index element={<ExtensionRegistration runtime={props.runtime} ims={props.ims} />} />
+                      <Route path={'create-return'} element={<CreateReturn runtime={props.runtime} ims={props.ims} />} />
+                      <Route path={'delete-order'} element={<DeleteOrder runtime={props.runtime} ims={props.ims} />} />
```
Comment (asalloum5) [inline admin-ui-sdk/banner-notification/order-view-button/web-src/src/components/App.js:38]:
```suggestion
                      <Route index element={<ExtensionRegistration runtime={props.runtime} ims={props.ims} />} />
```

File: admin-ui-sdk/order/custom-mass-action/.gitignore [hunk 4]
```diff
@@ -20,4 +20,3 @@ console.json
 .parcel-cache
 .DS_Store
 .vscode
-
```
Comment (asalloum5) [inline admin-ui-sdk/order/custom-mass-action/.gitignore:23]: can be reverted

Comment (mariam21arauj) [reply admin-ui-sdk/order/custom-mass-action/.gitignore:23]: Not sure why it still shows as if line 23 was removed. I fixed it.

File: admin-ui-sdk/banner-notification/order-view-button/README.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+## Features
+
+- **Custom Banner Notifications**: Add a custom Banner Notification for an order's 'Create Return' and 'Delete Order' buttons in the Commerce Admin panel to display a success or error message.
+
+## Prerequisites
+ 
+- Order custom view button extension point registered.
+- Adobe Commerce instance with IMS module installed and enabled.
+- Adobe Commerce Admin UI SDK installed and enabled.
```
Comment (ryanjcohen) [inline admin-ui-sdk/banner-notification/order-view-button/README.md:17]: I think mentioning that version 1.4.0+ is needed could be helpful here

File: admin-ui-sdk/banner-notification/order-view-button/package.json [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+        "@openwhisk/wskdebug": "^1.4.0",
+        "jest": "^29.4.2"
+    },
+    "scripts": {
+        "test": "jest --passWithNoTests ./test",
+        "e2e": "jest --collectCoverage=false --testRegex ./e2e"
+    },
+    "engines": {
+        
+    }
```
Comment (ryanjcohen) [inline admin-ui-sdk/banner-notification/order-view-button/package.json:42]: The empty `engines` doesn't add any additional info. Supported node versions could be specified like in the other examples

File: admin-ui-sdk/banner-notification/custom-mass-actions/web-src/src/components/ExtensionRegistration.js [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+governing permissions and limitations under the License.
+*/
+import { register } from '@adobe/uix-guest'
+
+export default function ExtensionRegistration() {
+    init().catch(console.error)
+}
+
+const init = async () => {
+    const orderExtensionId = 'order-custom-mass-action'
```
Comment (ryanjcohen) [inline admin-ui-sdk/banner-notification/custom-mass-actions/web-src/src/components/ExtensionRegistration.js:19]: Minor style comment - spacing here could be adjusted to match what's done elsewhere in the repo for style consistency https://github.com/adobe/adobe-commerce-samples/blob/main/admin-ui-sdk/menu/custom-menu/web-src/src/components/ExtensionRegistration.js

**Review by oshmyheliuk — APPROVED** [review]

**Pull request merged**